lib/flickr-transport.js

```diff
@@ -344,6 +363,19 @@ FlickrTransport.prototype.call = function (definition, params, additionalParams)
 					}
 				});
 
+				responseBody = responseBody || {};
+
+				// When we're requesting a token, it returns text that we need to parse
+				if (res.type === 'text/plain' && (operationId === "getRequestToken" || operationId === "getAccessToken")) {
```

Comment:
Are there any other `text/plain` operations? It might be nice to teach superagent how to handle `text/plain` with a [custom response body parser](http://visionmedia.github.io/superagent/#parsing-response-bodies).